src/aleph/vm/orchestrator/machine.py

```diff
@@ -0,0 +1,77 @@
+import json
+import re
+import subprocess
+from functools import lru_cache
+
+import psutil
+
+
+@lru_cache
+def get_hardware_info():
+    lshw = subprocess.Popen(["lshw", "-sanitize", "-json"], stdout=subprocess.PIPE, shell=False)
```

Comment:
Using `subprocess.Popen` would block the entire orchestrator since `.communicate()` is a synchronous blocking call. This should use https://docs.python.org/3/library/asyncio-subprocess.html instead.